internal/commands/run.go

```diff
@@ -72,6 +72,9 @@ func runCmd(dockerCli command.Cli) *cobra.Command {
 	cmd.Flags().StringVar(&opts.cnabBundle, "cnab-bundle-json", "", "Run a CNAB bundle instead of a Docker App")
 	cmd.Flags().StringArrayVar(&opts.labels, "label", nil, "Label to add to services")
 
+	//nolint:errcheck
+	cmd.Flags().SetAnnotation("cnab-bundle-json", "experimental", []string{"true"})
```

Comment:
should be `experimentalCLI` (on current codebase, I'm fine we use `experimental` but need to be consistent :P)

Reply:
Done [here](https://github.com/docker/app/pull/736/files)